src/server/constants.ts

```diff
@@ -43,7 +43,7 @@ export const GENERATE_TYPES_DETECT_ONE_TO_ONE_RELATIONSHIPS =
 
 export const DEFAULT_POOL_CONFIG: PoolConfig = {
   max: 1,
-  connectionTimeoutMillis: PG_CONN_TIMEOUT_SECS * 1000,
+  query_timeout: PG_CONN_TIMEOUT_SECS * 1000,
```

Comment:
A connection timeout is *not* a query timeout. imo it's very misleading to feed in a "CONN_TIMEOUT" here.

Reply:
Good catch, made a PR to revert it: https://github.com/supabase/postgres-meta/pull/777